packages/dev-console/src/ExtensionServerClient/types.ts

```diff
@@ -0,0 +1,224 @@
+/* eslint-disable @typescript-eslint/naming-convention, @typescript-eslint/no-namespace */
```

Comment:
This file is a lot more stuff to take in for users. Let's add docs in this PR, too

Reply:
Ok. I'll add it to the README file on the kit. I feel it's good to keep these docblocks here as they'll serve as reference for users using them as they show up with the IDE's autocomplete popup.